packages/mds-policy-service/jest.config.js

```diff
@@ -1 +1 @@
-module.exports = { ...require('../../jest.config.js') }
+module.exports = { ...require('../../jest.config.js'), setupFiles: ['./tests/configurePorts.ts'] }
```

Comment:
Any reason not to do this in the root config and just configure _all_ ports in one place? Would make it more obvious where and how to add new services.